Bottom: 5a8fdf922d33894eead201325c231d9bf02df0f7
Top:    bbedf18bb9333c0d03971463740c86015a1d19b3
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-15 10:56:34 -0700

Refresh of delete-swig-typemaps-for-file

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp b/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp
index 181a2e84083..70654a42384 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp
@@ -652,8 +652,10 @@ bool ScriptInterpreterPythonImpl::SetStdHandle(FileSP file_sp,
   PythonDictionary &sys_module_dict = GetSysModuleDictionary();
 
   auto new_file = PythonFile::FromFile(file, mode);
-  if (!new_file)
+  if (!new_file) {
+    llvm::consumeError(new_file.takeError());
     return false;
+  }
 
   save_file = sys_module_dict.GetItemForKey(PythonString(py_name));
